modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+= Production Readiness Checklist 
+
+Before running production workloads on Redpanda it’s worth ensuring that the cluster is deployed and configured according to best practices. The checklist in this guide will help you ensure that you are set up for success. In this guide we identify three levels of production readiness - level 1 through level 3. At level 3 you will be ready to run mission critical workloads in the largest of enterprises, whereas smaller organizations or less critical workloads may suffice at level 1.
+
+Redpanda provides multiple resources for deploying a production cluster. We recommend you utilize the Automated deployment instructions with the Redpanda Ansible collection. If you cannot use this method, we provide manual deployment instructions, which are more involved.
+
+== Checklists
+
+=== Level 1 checklist
+
+[%interactive]
+* [ ] <<redpanda-license>>
+* [ ] <<cluster-health>>
+* [ ] <<production-mode>>
+* [ ] <<redpanda-lmr>>
+* [ ] <<cpu-memory>>
+* [ ] <<mounted-disks>>
+* [ ] <<auth-enable>>
+* [ ] <<tls-enabled>>
+* [ ] <<redpanda-tuners>>
+* [ ] <<disk-perf>>
+* [ ] <<hostnames-interfaces>>
+* [ ] <<continuous-db>>
+* [ ] <<debug-bundle>>
+* [ ] <<topic-rf>>
+* [ ] <<maintenance-mode>>
+* [ ] <<decom-state>>
+
+
+
+=== Level 2 checklist
+
+[%interactive]
+* [ ] Dev / test environment available?
+* [ ] Metrics being captured and stored
+* [ ] System logs being captured and stored
+* [ ] Upgrade policy defined and implemented
+* [ ] Rack awareness configured correctly
+
+
+=== Level 3 checklist
+
+[%interactive]
+* [ ] Alerting configured for metrics
+* [ ] Backup or DR solution in place
+* [ ] Backup or DR solution tested
+* [ ] DR: is replication in place and is replication monitored
+* [ ] Is deployment automation used and could it be used to rebuild the cluster if required
+* [ ] Are audit logs being forwarded to an enterprise SIEM system
+
+
+== Level 1 Production Readiness
+
+[[redpanda-license]]
+=== Load Redpanda License
+
+Ensure that the Redpanda License has been loaded into the cluster configuration. This is required to enable Enterprise features described later. 
```

Comment:
add link on how to obtain redpanda enterprise license. Talk to sales?

Reply:
Linking to the licenses page should be sufficient.